proto/responses.proto

```diff
@@ -62,6 +62,20 @@ message SyncStateResponse {
     repeated NullifierUpdate nullifiers = 8;
 }
 
+message SyncNoteResponse {
+    // Number of the latest block in the chain
+    fixed32 chain_tip = 1;
+
+    // Block header of the block with the first note matching the specified criteria
+    block_header.BlockHeader block_header = 2;
+
+    // Proof for block header's MMR with respect to the chain tip.
+    merkle.MerklePath mmr_path = 3;
```

Comment:
I would add some more comments here to explain the context of the path since MMR proof consists of 3 elements (`forest`, `position`, `path`). In our case:

- `forest` is the same as `response.chain_tip + 1` (though would be good to double-check this).
- `position` is the same as `response.block_header.block_num`.
- `path` is the `response.mmr_path`.

Most of this should be almost self-explanatory, but doesn't hurt to spell this out explicitly.